### PR TITLE
docs: create reading lists

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Create reading lists for these types of users:
  - Newcomers
  - Intermediate
  - Advanced
- Short section "I don't know where to start"

## Context

Guide our readers through the relevant docs for their intended use. Basically the more customizations a user wants to make, the more they'll have to read. 🤓 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: docs/usage/.pages
Comment (HonkingGoose) [inline docs/usage/.pages:3]: By the way, this change will break a Cypress test on the Renovate docs repository. We have a test that checks if the next/previous buttons work properly. We hard-coded the `getting-started` page as the Next page.

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:61]:
```suggestion
- Define your own regex manager with [`regexManagers`](./configuration-options.md#regexmanagers)
- [Shareable config presets](./config-presets) 
```

Let's put shareable config presets in the list for "Advanced" users. It's one of the key selling points for Renovate, so we should probably include it.

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:24]:
```suggestion
- Use the Mend-hosted GitHub App, or let somebody else host Renovate for you
```

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:25]:
```suggestion
- Stick with the `config:base` (or the upcoming `config:best-practices` preset)
```

We did not merge this PR yet, so I'll call it the "upcoming preset".

- #16100 

Or I can drop the reference for now, if you want.

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:40]: Are you happy keeping it to just a list, or should I have some introductory text? Here's how the Beginners section could look:

> First, you'll have to install Renovate, read Installing & Onboarding to learn how.
> Once you've got Renovate up-and-running, check out your Dependency Dashboard, it shows all your pending updates.
> Then read about the Use Cases that people have for using Renovate.
>
> If you're having a problem with Renovate, read the Troubleshooting guide.
>
> Do you think Renovate is not responding, or merges updates slowly? First read the Known Limitations to learn what you can reasonably expect.
>
> And finally, we have Release notes for major versions of Renovate. These come with developer commentary, and tell you _why_ we're making certain changes.
>
> Whew, that was a lot of reading, but you're up to speed now.
> We hope you enjoy Renovate, and your updated dependencies!

Let me know what you want! 😉

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:62]:
```suggestion
- [Shareable config presets](./config-presets)
```

File: docs/usage/reading-list.md
Comment (rarkins) [reply docs/usage/reading-list.md:25]: I would prefer to avoid "upcoming", so either we merge this after it's available, or drop it now

File: docs/usage/.pages
Comment (viceice) [reply docs/usage/.pages:3]: can be fixed later

File: docs/usage/reading-list.md
Comment (viceice) [inline docs/usage/reading-list.md:62]:
```suggestion
- [Shareable config presets](./config-presets.md)
```

otherwise the link will probably be broken

**Review by viceice — COMMENTED** [review]
otherwise lgtm, we can improve later when we get user feedback

File: docs/usage/.pages
Comment (HonkingGoose) [reply docs/usage/.pages:3]: We'll have to fix it later when the docs change lands in the Renovate docs repository. I can't fix it now. 😄 

Now you know _why_ and _how_ the test is going to break, so you can fix it if I'm not around. 😉

File: docs/usage/reading-list.md
Comment (HonkingGoose) [reply docs/usage/reading-list.md:40]: I'll let @rarkins decide if they want a short list of things to read, or longer more descriptive paragraphs for each item. A longer text:

- risks duplicating content with other pages
- will probably be skimmed over by most readers anyway
- makes the text friendlier
- gives context for _why_ we want users to read certain pages

Comment (rarkins) [reply docs/usage/reading-list.md:40]: Short is fine

File: docs/usage/reading-list.md
Comment (HonkingGoose) [inline docs/usage/reading-list.md:26]:
```suggestion
- Use the Dependency Dashboard (`config:base` enables it automatically)
```

**Comment by HonkingGoose** [comment]
This is ready for final review and merge.

**Review by rarkins — APPROVED** [review]

**Pull request merged**
